applicationLibrary/molecularDynamics/molecularDynamicsLibrary/LJFunctor.h

```diff
@@ -670,8 +670,8 @@ class LJFunctor : public autopas::Functor<Particle, LJFunctor<Particle, applyShi
       _potentialEnergySum /= 6.;
       _postProcessed = true;
 
-      AutoPasLog(TRACE, "Final potential energy {}", _potentialEnergySum);
-      AutoPasLog(TRACE, "Final virial           {}", _virialSum[0] + _virialSum[1] + _virialSum[2]);
+      AutoPasLog(INFO, "Final potential energy {}", _potentialEnergySum);
+      AutoPasLog(INFO, "Final virial           {}", _virialSum[0] + _virialSum[1] + _virialSum[2]);
```

Comment:
Sorry, I never actually submitted my comments. These were all just pending!

> If we are actually calculating the globals then we probably want to know what the globals are.
> 
> Potentially, this gets too spammy, in which case a separate csv logger might be more appropriate. This should be left to a future PR. I would accept changing this to `DEBUG` for now as a temporary solution if somebody expects keeping this at `INFO` to be a problem.
> 
> Comment stands for other functors.

However, also see my discussion with Markus:

> > > As md-flexible is designed as an example of AutoPas, we are not interested in temperature as a quantity of interest. Rather, we are interested in the temperature to ensure that experiments designed as test cases of AutoPas apply sensible thermostat settings (for example, checking that Delta Temp is not set so low that the temperature never reduces to the target temperature. As such, these logs are at the DEBUG level, not the INFO level.
> > 
> > 
> > Wouldn't the same reasoning apply to the quantities of global energy and virial? Tbh, I'm also a bit undecided here, as I think logging of global energy and virial should in theory be a functionality of the simulator. But then maybe you don't want to log it every timestep and suddenly everything becomes more involved... So I think I would be fine with your solution for now
> 
> This is a fair point. I would be okay with this as a `DEBUG` level message.


I think `DEBUG` is probably suitable for now under the idea that md-flexible is primarily an example and so the point of compiling with globals is to test performance implications and so we don't actually want the results at `INFO` level. It may be useful in the future (not in this PR) to provide globals (+temperature/kinetic energy/total energy) logging as this is a useful way to tell if simulations designed to test performance are still physically sound.

Reply:
> If we are actually calculating the globals then we probably want to know what the globals are.

Counterpoint: If I'm calculating the globals I want to see the impact it has on runtime.

I agree that if someone actually wants to have the values we should include them maybe in some logger. Remember also that outputs that happen every iteration break the progress bar.

Reply:
Made `DEBUG` in 9e0d2637